esp/src/src-react/components/Files.tsx

```diff
@@ -30,6 +30,9 @@ const FilterFields: Fields = {
 };
 
 function formatQuery(filter) {
+    if (filter.Index) {
+        filter.ContentType = "key";
```

Comment:
You should also `delete filter.Index` as well.

Reply:
If filter.Index is removed, the checkbox will not show as checked if you were to land on the url & then open the Filter to modify it. Changing the name of the field to "ContentType" also has the same issue. If the value is correct for the request the checkbox won't be checked when repopulated via the url, since it is expecting a boolean.

Reply:
What I do in the `workunits.tsx` component is:

```
function formatQuery(_filter) {
    const filter = { ..._filter };
  ...
    return filter;
}
```

Reply:
Thanks Gordon. Changed as requested & also rebased.